lumigator/python/mzai/jobs/inference/inference_config.py

```diff
@@ -115,8 +116,24 @@ class SamplingParameters(BaseSamplingParameters):
     model_config = ConfigDict(extra="forbid")
 
 
+class AutoTokenizerConfig(BaseAutoTokenizerConfig, arbitrary_types_allowed=True):
+    """Settings passed to a HuggingFace AutoTokenizer instantiation."""
+
+    model_uri: AssetPath = Field(title="The Model HF Hub repo ID", exclude=True)
+    model_max_length: int
+    use_fast: bool = True
+    trust_remote_code: bool = False
```

Comment:
Should we add a `truncation` argument here, with the default set to `True`?

Reply:
I would move this to the `AutoTokenizerConfig` class because it is a tokenizer-specific argument, isn't it? What do you think?